Description: Compatibility fixes for ICU 65.
Bug-Debian: https://bugs.debian.org/952156
Origin: upstream, commit:6c7defb
Last-Update: 2020-06-11
---

--- gnustep-base.orig/Tests/base/NSLocale/general.m
+++ gnustep-base/Tests/base/NSLocale/general.m
@@ -22,6 +22,7 @@
 
   // These tests don't really work all that well.  I need to come up with
   // something better.  Most of the ones that fail are because nil is returned.
+  testHopeful = YES;
   locale = [[NSLocale alloc] initWithLocaleIdentifier: @"es_ES_PREEURO"];
   PASS_EQUAL([locale objectForKey: NSLocaleIdentifier],
     @"es_ES@currency=ESP",
@@ -64,7 +65,6 @@
     "NSLocaleCurrencyCode key returns 'ESP'");
   PASS_EQUAL([locale objectForKey: NSLocaleCollatorIdentifier],
     @"es_ES@currency=ESP", "NSLocaleCollatorIdentifier for Spain");
-  testHopeful = YES;
   PASS_EQUAL([locale objectForKey: NSLocaleGroupingSeparator],
     @".",
     "NSLocaleGroupingSeparator key returns '.'");
